zyte_api/aio/client.py

```diff
@@ -18,9 +18,8 @@
 from ..stats import AggStats, ResponseStats
 from ..utils import user_agent
 
-AIO_API_TIMEOUT = aiohttp.ClientTimeout(total=API_TIMEOUT + 60,
-                                        sock_read=API_TIMEOUT + 30,
-                                        sock_connect=10)
+
+AIO_API_TIMEOUT = aiohttp.ClientTimeout(total=max(API_TIMEOUT, 60) * 5)
```

Comment:
This matches by default the default aiohttp. 5 minutes of timeout seems reasonable to me. 

I've removed `sock_read` and `sock_connect`. Having lower values for them will improve interactivity, but can cause timeouts under heavy CPU load. I prefer having a global total timeout.